Source/Core/Core/ConfigManager.cpp

```diff
@@ -726,6 +731,67 @@ void SConfig::LoadSettingsFromSysconf()
   bPAL60 = sysconf.GetData<u8>("IPL.E60") != 0;
 }
 
+void SConfig::ResetRunningGameMetadata()
+{
+  SetRunningGameMetadata("00000000", 0, 0);
+}
+
+void SConfig::SetRunningGameMetadata(const DiscIO::IVolume& volume)
+{
+  u64 title_id = 0;
+  volume.GetTitleID(&title_id);
+  SetRunningGameMetadata(volume.GetGameID(), title_id, volume.GetRevision());
+}
+
+void SConfig::SetRunningGameMetadata(const IOS::ES::TMDReader& tmd)
+{
+  const u64 title_id = tmd.GetTitleId();
+  std::string game_id;
+
+  if (IOS::ES::IsDiscTitle(title_id))
+  {
+    const u32 title_identifier = Common::swap32(static_cast<u32>(title_id));
+    const u16 group_id = Common::swap16(tmd.GetGroupId());
+
+    char ascii_game_id[6];
+    std::memcpy(ascii_game_id, &title_identifier, sizeof(title_identifier));
+    std::memcpy(ascii_game_id + sizeof(title_identifier), &group_id, sizeof(group_id));
+
+    game_id = ascii_game_id;
+  }
+  else
+  {
+    game_id = StringFromFormat("%016" PRIX64, title_id);
+  }
+
+  SetRunningGameMetadata(game_id, title_id, tmd.GetTitleVersion());
+}
+
+void SConfig::SetRunningGameMetadata(const std::string& game_id, u64 title_id, u16 revision)
+{
+  const bool was_changed = m_game_id != game_id || m_title_id != title_id || m_revision != revision;
+  m_game_id = game_id;
+  m_title_id = title_id;
+  m_revision = revision;
+
+  if (was_changed)
+  {
+    NOTICE_LOG(BOOT, "Game ID set to %s", game_id.c_str());
```

Comment:
Well, that's only because CORE didn't exist. Until one week ago, and it is only in master since <5 minutes ago :P 

I don't really care either way, so feel free to ignore.